docs/customizing.md

```diff
@@ -0,0 +1,238 @@
+Customizing Your Pulp Deployment
+================================
+
+Background info on Ansible variables
+------------------------------------
+
+Because the Pulp Installer is based on Ansible, it inherits two Ansible designs:
+
+1. Variables can be set in many different ways by the user.
+2. Each role has a set of variables it recognizes for customizing the deployment.
+
+This section (and the docs in general) will not cover every possible way of [setting variables](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#where-to-set-variables), but instead cover the
+simplest way: in the Ansible "playbook" itself.
+
+This means that the playbook, like listed in [Getting Started](quickstart.md#example-playbook-for-installing-plugins), has a section `vars:`.
+
+This section intends to demonstrate how to use Ansible variables to configure Pulp rather than listing every variable. Documentation for each Ansible variable can be found in each [Ansible role's README file.](index.md#roles)
+
+Note that [variables are YAML](https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#yaml-basics) and have types such as dictionaries (key-value pairs), lists, strings and booleans.
+
+(JSON can also be used as values to the variables.)
+
+Some of the installer's variables, `pulp_settings` and `pulp_install_plugins` in particular, are dictionaries with prescribed variables or prescribed nested dictionaries nested under them. So do not be surprised when you see syntax like the following:
+```
+  vars:
+    pulp_install_plugins:
+       pulp-container: {}
+       pulp-deb:
+         upgrade: true
+      pulp-file:
+        collectstatic: true
+        version: "9.8.7"
+```
+
+What that means **in terms of YAML syntax** is:
+* There is a dictionary named `pulp_install_plugins` whose value is 3 nested dictionaries, `pulp-deb`, `pulp-file` and `pulp-container`.
+* There is a nested dictionary named `pulp-container` dictionary, whose value is an empty dictionary `{}`.
+* There is a nested dictionary named `pulp-deb`, whose value is is 1 key-value pair.
+* There is a nested dictionary named `pulp-file`, whose value is 2 key-value pairs.
+
+See [the section below](#example-of-setting-a-multitude-of-variables) for more info on what this means in terms of actual Pulp Installer behavior.
+
+
+Pulp Installer's types of Variables
+-----------------------------------
+Generally, there are 3 categories of variables
+1. Those that make pulp_installer compatible with your environment. For example, the firewall.
+2. Preferences for how Pulp is configured and deployed. In particular, which plugins to install.
+3. Variables to control which version of plugins are installed, for reproducibility.
```

Comment:
```suggestion
3. Variables to control which version of the individual plugins are installed.
```
I think control is the big thing here. Even for the first run, so imho it's more about predictability than reproducibility.
Feel free to disregard.